### ISSUE TITLE
bb-admin not there

### ISSUE THREAD

**Issue by sacko1966** [boxbilling/boxbilling#1084]
Hi I would love to take a bash at this but i have installed it its all working well the client side is but the admin side login dont exist 
/billing/bb-admin/staff/login

Not Found
The requested URL was not found on this server.

bb-admin is not in the boxbilling.zip install directory 

where is it lol
Thanks

**Comment by evrifaessa** [comment]
> oh and what you said above was already set
Yes it was because I did set them. But after a clean installation, you have to run these before using BB for the first time.
